docs/_data/help.yaml

```diff
@@ -116,6 +116,18 @@
       Although we encourage you to use an editor such as **Visual Studio Code** to edit the forms.yaml file, preferably with some form of source control.  Ansible Forms comes with an internal yaml designer.  
       And although only admins can see the designer, you might want to disable it completely by setting this variable to 0.
     version: 4.0.0
+  - name: USE_YTT
+    type: number
+    choices:
+      - name: 0
+        description: Does not execute ytt on the forms.yaml
+      - name: 1
+        description: Executes ytt on the forms.yaml
+    default: 0
+    short: Enable the ytt interpreter
+    description: |
+      https://github.com/carvel-dev/ytt is a tool for yaml templating. It can be activated by this variable.
+    version: 5.0.1
```

Comment:
Nice, thank you!  
I assumed you are using the devel branch, but main is the default branch... I was not completely sure which branch to target.

Reply:
ps: your are the first one making decent code suggestions.  I can use someone to team up with.  think about it.

Reply:
Currently, I am working on a project, where we integrate ansibleforms for operations. Within the scope of the project, I am happy to contribute.  
For example, we need OIDC integration because we are using keycloak as an identity provider.
I already started working on this feature and once done, I will create a PR.  
  
I am not really active on GitHub and I can't see an option to message you directly, so I just put my words here:  
  
Kudos to you for this project. It is exactly what we need for our operations concept. We were struggling with AWX for quite some time now. We used a custom version of the development docker compose deployment. This worked but AWX is completely lacking any dynamic behaviour in the fronted, which makes it just not usable for us.  
AnsibleForms fills this gap and provides all the functionalities we need 👍 

Some words on the project I am working, as you probably like what we are doing ;-)  
AnsibleForms is going to space!  
Together with ESA (European Space Agency) we are developing a new infrastructure for the ISS Columbus Module.
AnsibleForms will be used by the operators to configure the infrastructure during daily operations (e.g. manage bandwidth per user). Yesterday, the final decision was taken to throw AWX away and use AnsibleForms.

Reply:
very cool. 

https://www.linkedin.com/in/mirkovancolen?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app